website/src/components/broadcast/PlayerHero.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="hero-card d-flex flex-column flex-center" v-if="hero">
+    <div class="hero-name d-flex flex-row text-center font-weight-bold"><span v-if="roleIcon" class="icon" v-html="roleIcon"></span> {{heroName}}</div>
+    <div class="hero-role d-flex flex-row text-center">{{heroRole}}</div>
+    <div class="hero-portrait d-flex flex-row flex-center"><img :src="heroPortraitURL"/></div>
+  </div>
+</template>
+
+<script>
+
+export default {
+    name: "PlayerHero",
+    props: ["hero"],
+    computed: {
+        heroName() {
+            if (!this.hero?.name) return null;
+            return this.hero.name;
+        },
```

Comment:
Can be simplified. You don't even need the `|| null` really, it will just force anything falsy to be `null` instead.
```suggestion
        heroName() {
            return this.hero?.name || null;
        },
```

Reply:
done in af0f00d